implementations/micrometer-registry-wavefront/build.gradle

```diff
@@ -1,6 +1,7 @@
 dependencies {
     compile project(':micrometer-core')
     compile 'org.slf4j:slf4j-api:1.7.+'
+    compile 'com.wavefront:wavefront-sdk-java:1.2'
```

Comment:
Let's make this `latest.release` so as we move locks we pick the latest versions.